pkg/roles/roles.go

```diff
@@ -56,6 +56,12 @@ type RoleProvision struct {
 	OptionalMembers []string `json:"optional_members"`
 }
 
+type RoleRes struct {
```

Comment:
Please remove this struct we are using `MemberRoleActions` instead of this struct